templates/XStepExplanation/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Centered from '../chromes/Centered'
+import * as List from '../../List'
+
+import classNamesBind from 'classnames/bind'
+import defaultStyles from './styles.scss'
+
+const baseClass = 'three-step-explanation'
+
+const classes = {
+  list: `${baseClass}__list`
+}
+
+const classNames = classNamesBind.bind(defaultStyles)
+
+export default function XStepExplanation ({
```

Comment:
👍 

The thing is that we don't support onSwipe anywhere currently and I didn't want to commit to having that with the fix deadline. The Sketch has an example that is closer to the one you show with several steps (although white background) but we scoped it out for the time being.

Worst case scenario, we can do this next drop.